aggsender/query/certificate_query.go

```diff
@@ -76,18 +77,18 @@ func (c *certificateQuerier) GetLastSettledCertificateToBlock(
 			cert.NewLocalExitRoot.String(), err)
 	}
 
-	// TODO - this might need to be changed once agglayer gives support for this
 	// 2. Get the latest settled imported bridge exit block number
-	latestSettledIbeGlobalIndex, bridgeExitHash, err := c.agglayerClient.GetLatestSettledImportedBridgeExit(ctx)
+	networkStatus, err := c.agglayerClient.GetNetworkStatus(ctx, cert.NetworkID)
 	if err != nil {
 		return 0, fmt.Errorf("failed to get latest settled imported bridge exit from agglayer: %w", err)
 	}
 
-	if latestSettledIbeGlobalIndex != nil {
-		lastImportedBridgeExitBlock, err = c.getBlockNumFromGlobalIndex(ctx, latestSettledIbeGlobalIndex, bridgeExitHash)
+	if networkStatus.SettledImportedBridgeExit != nil {
+		lastImportedBridgeExitBlock, err = c.getBlockNumFromGlobalIndex(
+			ctx, networkStatus.SettledImportedBridgeExit.GlobalIndex, networkStatus.SettledImportedBridgeExit.BridgeExitHash)
 		if err != nil {
 			return 0, fmt.Errorf("failed to resolve the block number for last imported bridge exit %s: %w",
-				latestSettledIbeGlobalIndex.String(), err)
+				networkStatus.SettledImportedBridgeExit.GlobalIndex.String(), err)
 		}
```

Comment:
Nitpick, slightly more readable:
```suggestion
	settledIBE := networkStatus.SettledImportedBridgeExit
	if settledIBE != nil {
		lastImportedBridgeExitBlock, err = c.getBlockNumFromGlobalIndex(
			ctx, settledIBE.GlobalIndex, settledIBE.BridgeExitHash)
		if err != nil {
			return 0, fmt.Errorf("failed to resolve the block number for last imported bridge exit %s: %w",
				settledIBE.GlobalIndex.String(), err)
		}
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/913/commits/88047617be5bec9b62816b8fddf43a1056601e99